src/lib/select/select.md

```diff
@@ -92,3 +92,11 @@ Here are the available global options:
 - <kbd>DOWN_ARROW</kbd>: Focus next option
 - <kbd>UP_ARROW</kbd>: Focus previous option
 - <kbd>ENTER</kbd> or <kbd>SPACE</kbd>: Select focused item
+
+### Accessibility
+The select component without text or label should be given a meaningful label via
+`aria-label` or `aria-labelledby`.
+
+The select component has `role="listbox"` and options inside select has `role="option"`.
+Autocomplete trigger sets `aria-owns` to the autocomplete's id, and sets `aria-activedescendant` to
```

Comment:
This one talks about autocomplete instead of select. Also select doesn't use `aria-activedescendant`, but it moves focus between the options.